src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SynthesizedContainer : NamedTypeSymbol
     {
+        internal const string StructFieldCaptureName = "<struct-field-capture-name>";
```

Comment:
thanks!

---
In reply to: [134594419](https://github.com/dotnet/roslyn/pull/21664#discussion_r134594419) [](ancestors = 134594419)